### PR TITLE
Add testwiki stuff

### PR DESCRIPTION
**Pull request by Reception123**
add here so i can be sure its correct

### CODE REVIEW

File: Miraheze/Mirahezelocalset.php [hunk 1]
```diff
@@ -405,6 +405,16 @@
 			'superadmin' => array(
 				'admin',
 			),
```
Comment (NDKilla) [inline Miraheze/Mirahezelocalset.php:407]: When you add this here it prevents the quantix wiki array above from being closed. Add another ), in the right spot above

File: Miraheze/Mirahezelocalset.php [hunk 2]
```diff
@@ -502,6 +512,11 @@
 		'+testwiki' => array(
 			'bureaucrat' => array(
 				'bureaucrat' => true,
```
Comment (NDKilla) [inline Miraheze/Mirahezelocalset.php:513]: The bureaucrat array isn't closed

**Comment by Reception123** [comment]
@NDKilla added the array. Any other problem?

File: Miraheze/Mirahezelocalset.php [hunk 3]
```diff
@@ -502,6 +512,11 @@
...
+			'consul' => array(
+				'read' => true,
+				'consul' = true,
+					),
+				),
```
Comment (NDKilla) [inline Miraheze/Mirahezelocalset.php]: This array is closed 3 times because of where you copy pasted..

File: Miraheze/Mirahezelocalset.php [hunk 4]
```diff
@@ -404,6 +404,16 @@
 			),
 			'superadmin' => array(
 				'admin',
+	),		),
```
Comment (NDKilla) [inline Miraheze/Mirahezelocalset.php]: Please move these two ), to separate lines and align them with the arrays they close

**Comment by Reception123** [comment]
Already done in orginial LS plus has conflicts.

**Pull request closed without merging**